docs/src/content/docs/fr/environmental-impact.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Documents écologiques
+description: Découvrez comment Starlight peut vous aider à créer des sites de documentation plus écologiques et à réduire votre empreinte carbone.
```

Comment:
Is saying "des sites de documentation" is really necessary ? We can replace that by "des documentations", no ?

Reply:
Yes!